src/components/ColorRampDropdown.tsx

```diff
@@ -23,12 +23,44 @@ const ColorRampSelector: React.FC<ColorRampSelectorProps> = (propsInput): ReactE
   const props = { ...defaultProps, ...propsInput } as Required<ColorRampSelectorProps>;
   const theme = useContext(AppThemeContext);
 
+  // Force the dropdown to stay open when clicked for accessibility. Close it again
+  // when focus is lost.
+  const [forceOpen, setForceOpen] = useState(false);
+  const componentContainerRef = React.useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (!forceOpen) {
+      return;
+    }
+    const doesContainTarget = (target: EventTarget | null): boolean => {
+      return (
+        (target instanceof Element &&
+          componentContainerRef.current &&
+          componentContainerRef.current.contains(target)) ||
+        false
+      );
+    };
+    // Handle focus loss for tab navigation
+    const handleFocusLoss = (event: FocusEvent): void => {
+      if (!doesContainTarget(event.relatedTarget)) {
+        setForceOpen(false);
+      }
+    };
+
+    componentContainerRef.current?.addEventListener("focusout", handleFocusLoss);
+    return () => {
+      componentContainerRef.current?.removeEventListener("focusout", handleFocusLoss);
+    };
+  }, [forceOpen]);
+
```

Comment:
This is nice! Had a bit of trouble understanding the logic at first, but it makes the component feel like it "just works" for multiple input methods. I might at least mess around with making it a custom hook since it's also used in `LabeledDropdown`, but honestly I'm leaning towards it being fine to live in two places.

Reply:
I'll write a TODO to potentially refactor it into a hook later, but I'll leave it for now. Thanks for the suggestion!

Reply:
Also improved the comments here, sorry that they were unclear!